.travis.yml

```diff
@@ -5,12 +5,7 @@ language: java
 jdk:
   - oraclejdk8
 
-addons:
-  apt:
-    packages:
-      - oracle-java8-installer
-      - oracle-java8-set-default
+script: ./gradlew clean build
```

Comment:
@andrii-kovalenko-ct yes, I know. But I left it to be more explicit that it's a clean slate.